spacy_llm/tasks/ner.py

```diff
@@ -53,9 +53,12 @@ def make_ner_task_v2(
     labels_list = split_labels(labels)
     raw_examples = examples() if callable(examples) else examples
     span_examples = [SpanExample(**eg) for eg in raw_examples] if raw_examples else None
+    if template is None:
+        raise ValueError("A template must be supplied. It cannot be 'None'.")
+    task_template = template() if callable(template) else template
```

Comment:
IMO `template` should be resolved by the config system (if e. g. `spacy.FileReader.v1` is set), so I don't think we ever get a `Callable` here?

Reply:
fixing this

Reply:
Ok I removed this and also just remove the error if template is None. We have type hints here for a reason + we have a default template for each task so the user would need to explicitly set the template to None